effect_subject/models/health_economics.py

```diff
@@ -1,9 +1,262 @@
+from django.db import models
+from edc_constants.choices import YES_NO
+from edc_constants.constants import NOT_APPLICABLE
 from edc_model import models as edc_models
 
+from ..choices import CHILDCARE_CHOICES, PAYEE_CHOICES, TRANSPORT_CHOICES
 from ..model_mixins import CrfModelMixin
 
 
 class HealthEconomics(CrfModelMixin, edc_models.BaseUuidModel):
+    buy_refill_drug = models.CharField(
+        verbose_name="Over the past period since your treatment, did you buy any drugs or had drug refill?",
+        max_length=15,
+        choices=YES_NO,
+    )
+
+    amount_spent_antiretroviral_drugs_past = models.IntegerField(
+        verbose_name="How much did you spend on Antiretroviral drugs",
+    )
+
+    amount_spent_other_drugs_past = models.IntegerField(
+        verbose_name="How much did you spend on Other drugs for relief",
+    )
+
+    payment_method_antiretroviral_drugs_past = models.CharField(
+        verbose_name="If YES, how did you pay or who paid for Antiretroviral drugs?",
+        max_length=45,
+        choices=PAYEE_CHOICES,
+        default=NOT_APPLICABLE,
+    )
+
+    payment_method_other_drugs_past = models.CharField(
+        verbose_name="If YES, how did you pay or who paid for Antiretroviral drugs?If YES, how did you pay or "
+        "who paid for Other drugs for relief?",
+        max_length=45,
+        choices=PAYEE_CHOICES,
+        default=NOT_APPLICABLE,
+    )
+
+    spent_money_other_health_activities_past = models.CharField(
+        verbose_name="Since your last study visit, did you spend money on other activities (not drugs) relating to "
+        "your health?",
+        max_length=15,
+        choices=YES_NO,
+    )
+
+    other_health_activities_past = models.CharField(
+        verbose_name="If YES, what was the activity?",
+        max_length=145,
+        blank=True,
+        null=True,
+    )
+
+    amount_spent_other_health_activities_past = models.IntegerField(
+        verbose_name="If YES, how much was spent on other activities (not drugs) relating to your health?",
+    )
+
+    payment_method_other_health_activities_past = models.CharField(
+        verbose_name="If YES, how did you pay or who paid for these activities?",
+        max_length=45,
+        choices=PAYEE_CHOICES,
+        default=NOT_APPLICABLE,
+    )
+
+    num_day_activities_disrupted = models.IntegerField(
+        verbose_name="Over the past period since your treatment, how many days were your normal activities disrupted "
+        "through illness?",
+    )
+
+    amount_spent_healthcare_last_month = models.IntegerField(
+        verbose_name="How much in total has been spent on your healthcare in the last month?"
+    )
+
+    activities_not_come_clinic = models.CharField(
+        verbose_name="What would you be doing in your life, if you had not come to the hospital or clinic",
+        max_length=45,
+        choices=CHILDCARE_CHOICES,
```

Comment:
Childcare choices has NA option, which I don't think is applicable for this question.  I suggest creating a separate set of choices, with NA omitted.